src/libraries/System.Security.Permissions/src/System.Security.Permissions.csproj

```diff
@@ -5,6 +5,9 @@
     <ExcludeCurrentNetCoreAppFromPackage>true</ExcludeCurrentNetCoreAppFromPackage>
```

Comment:
Both the src-proj and the ref-proj say that netcoreappcurrent is excluded, and I don't see a marker saying this is an inbox package.

Maybe I don't understand the infra enough, but it feels like this change is a no-op, since it only affects non-distributed content.  (Perhaps all that's required is clearing the ExcludeCurrentNetCoreAppFromPackage variables?)

Reply:
Yeah we should remove `ExcludeCurrentNetCoreAppFromPackage ` in order to have the latest asset in the package... we will have to bring it back when `NetCoreAppCurrent` becomes net6.0 -- but that will be done as part of that change.

The reason why we need to exclude it now is because with this the netcoreapp3.0 and net5.0 impl and ref assemblies now differ and we need to have both assets in the package.

Reply:
You're right. I pinged @safern offline and your proposal appears to be correct. I'll send an update.